.github/pull_request_template.md

```diff
@@ -13,5 +13,6 @@ This PR....
 * [ ] The samples added / modified have been fully tested.
 * [ ] Workflow files have been added / modified, if applicable.
 * [ ] Region tags have been properly added, if new samples.
+* [ ] Editable variables have been used, where applicable.
```

Comment:
**Not** necessarily in this PR, but it would be useful to hyperlink the various requirements in the template to link to specific, e.g. anchors of the contributing file, so that a new contributor seeing "editable variables" can click on it if they don't know what that is / how to use it

Reply:
Sounds good.
Marking as resolved since this is out-of-scope for this PR.